plugins/Dispersion/Dispersion.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Dispersion.h
+ *
+ * Copyright (c) 2023 Lost Robot <r94231/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+
+#ifndef DISPERSION_H
+#define DISPERSION_H
+
+#include "Effect.h"
+#include "DispersionControls.h"
+#include "lmms_math.h"
+#include <qmath.h>
+
+namespace lmms
+{
+
+constexpr int MAX_DISPERSION_FILTERS = 400;
```

Comment:
Just curious, what defines this limit?

Reply:
It was chosen arbitrarily.  I intended to increase it to 999 later on but forgot to, thanks for pointing it out.